wob.c

```diff
@@ -230,11 +231,17 @@ wob_destroy(struct wob *app)
 	wl_display_disconnect(app->wl_display);
 }
 
+/*
+Input format:
+percentage bgColor borderColor barColor
+25 #FF000000 #FFFFFFFF #FFFFFFFF
+*/
 static bool
-wob_parse_input(const char *input_buffer, uint16_t *percentage)
+wob_parse_input(const char *input_buffer, uint16_t *percentage,
```

Comment:
I think logic around redrawing static should be kept outside this function. You can check if `background_color != old_background_color` or `old_border_color != border_color` and redraw it only in this case.

Reply:
I was trying to avoid using the memory to keep track of the old colors but it's literally four bytes per color so I'm just a bit crazy. I'll get on it now.